rules/60-sensor.rules

```diff
@@ -7,4 +7,8 @@ SUBSYSTEM=="iio", KERNEL=="iio*", SUBSYSTEMS=="usb|i2c", \
   IMPORT{builtin}="hwdb 'sensor:modalias:$attr{modalias}:$attr{[dmi/id]modalias}'", \
   GOTO="sensor_end"
 
+DRIVERS=="cmpc_accel_v4", \
```

Comment:
> Here it walks up the chain as well

$attr{modalias} doesn't walk up the chain in strictly the same way that SUBSYSTEMS= would look for a match. From the man page:

> If the matching device does not have such an attribute, and a previous KERNELS, SUBSYSTEMS, DRIVERS, or ATTRS test selected a parent device, then the attribute from that parent device is used.

So in this case, when looking for event7's modalias, it doesn't visit it's immediate parent (input8) but instead goes straight to the grandfather ACCE0001:00 which was selected by SUBSYSTEMS. That's why the rule in my last comment works.

> ${attr[XXX]modalias} where XXX is an attribute that's unique to the ACPI device.

It doesn't work and I checked the udev source and I don't think $attr{[XXX]modalias} works like this. This type of syntax is handled by `util_resolve_subsys_kernel` expects `XXX` to be in the format: `subsystem/sysname` (so a value of just "hid" is invalid). Then instead of walking from the device up through it's ancestors, it instead starts at the root of /sys and looks up the subsystem and the sysname beneath that.